Formula/s/sleek-sql.rb

```diff
@@ -0,0 +1,20 @@
+class SleekSql < Formula
```

Comment:
why not just name the formula as `sleek`?

Reply:
I am ok with that.  There is a cask with the same name that is completely different.   I don't know what the policy is between casks and formulas.

Reply:
it is okay, we have lots of overlaps between casks and formulae.

Reply:
I just realized you already mentioned it in the PR comment, yeah, it is okay for the name conflict btw casks and formulae, but not okay with the formulae (also if the artifacts conflict, it would need conflicts_with on the formulae side as well)